CONTRIBUTING.md

```diff
@@ -54,6 +54,20 @@ see [ABI Policy Guide](sycl/doc/ABIPolicyGuide.md) for more information.
 - Run regression tests -
 [instructions](sycl/doc/GetStartedGuide.md#test-dpc-toolchain).
 
+**Guidelines for adding Clang FE tests**:
+- Use `sycl::` namespace instead of `cl::sycl::`
+
+- Include sycl mock headers as system headers.
+Example: `-internal-isystem %S/Inputs`
+`#include "sycl.hpp"`
+
+- Use SYCL functions for invoking kernels from the mock header `(single_task, parallel_for)`
```

Comment:
Why parallel_for_work_group is missing in the list?

Reply:
I missed adding "etc" at the end, but I can update it to include `parallel_for_work_group` since that's the only one missing from the list.